packages/android_alarm_manager_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0
+
+ - **FEAT**: we can now send extra data to alarm manager and receive it in our callback (#1014).
```

Comment:
Wouldn't it be nicer to have a link?
```suggestion
 - **FEAT**: we can now send extra data to alarm manager and receive it in our callback ([#1014](https://github.com/fluttercommunity/plus_plugins/pull/1014)).
```

Reply:
I mean, we could do it, but I am not going to generate links for every single changelog generated by melos